__tests__/apiTests.test.js

```diff
@@ -109,4 +109,59 @@ describe("GET /api/articles",()=>{
                 })
         })
     })
+})
+
+describe("GET /api/articles/:article_id/comments",()=>{
+    test("get status code 200, and return all the comments for the specified article",()=>{
+        return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+            .then(({body})=>{
+                const comments = body.comments
+                expect(comments.length).toBeGreaterThan(0)
+                comments.forEach((comment)=>{
+                    const commentParts = Object.values(comment)
+                    expect(Object.keys(comment)).toMatchObject(["comment_id","body","article_id","author","votes","created_at"])
```

Comment:
this defeats the purpose of using .toMatchObject in my opinion to just assert against an array, and then use typeof for the next expect which is quite hard to read